nevergrad/benchmark/experiments.py

```diff
@@ -77,6 +100,21 @@ def naivemltuning(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     for xp in internal_generator:
         yield xp
 
+# We register only the sequential counterparts for the moment.
+@registry.register
+def seqkerastuning(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Sequential counterpart of mltuning."""
+    internal_generator = kerastuning(seed, overfitter=False, seq=True)
+    for xp in internal_generator:
+        yield xp
+
+# We register only the sequential counterparts for the moment.
+@registry.register
+def naiveseqkerastuning(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
```

Comment:
function names can containt `_`, i would tend to use it for ease of reading, even though the old mltuning xps did not. This is becoming a big long to parse :s